frigate/http.py

```diff
@@ -539,6 +540,16 @@ def events():
     if not has_snapshot is None:
         clauses.append((Event.has_snapshot == has_snapshot))
 
+    if time_range != DEFAULT_TIME_RANGE:
+        times = time_range.split(",")
+        time_before = times[0]
+        time_after = times[1]
+
+        start_hour_fun = fn.strftime(
+            "%H:%M", fn.datetime(Event.start_time, "unixepoch", "localtime")
+        )
+        clauses.append((start_hour_fun < time_before and start_hour_fun > time_after))
```

Comment:
I tried using `clauses.append((time_after < start_hour_fun < time_before))` but it didn't respect `time_after <`

Reply:
Actually further testing and this didn't work